### ISSUE TITLE
sponsored proxy

### ISSUE THREAD

**Issue by ahad-script** [p1ratrulezzz/mtproxy-server-linux#7]
How can i add my telegram channel for sponsored mtproto proxy?

**Comment by p1ratrulezzz** [comment]
I've already asked the same question in https://github.com/danog/MadelineProto/issues/429#issuecomment-391290124 . Looks like it is not possible yet

**Comment by ahad-script** [comment]
This mtproto proxy has sponsored channel:
https://t.me/proxy?server=dqipp.tgproxy.network&port=8443&secret=726564736869656c6476706e2e636f6d

**Comment by p1ratrulezzz** [comment]
Yeah, all these with tgproxy domain have sponsor channel. Looks like these come from telegram server, not the proxy. And also its impossible to find out what software they are using on these servers. I've searched a lot and saw few similar questions everywhere. Just be pationate and wait until some official documentation will be published, then all these features will be added to all popular mtproxy implementations

**Comment by p1ratrulezzz** [comment]
https://github.com/makkarpov/mtoxy/issues/1 also some information research from developer of Java implementation

**Comment by p1ratrulezzz** [comment]
https://github.com/danog/MadelineProto/issues/429 Read information here. So currently it is not implemented in none of mtproto proxy servers yet (except the official one).
